group_12/compress.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from sklearn.cluster import KMeans

```

Comment:
These libraries should also be added to the dependency list.

Reply:
I will add these later, thank you!